src/authorization/graphql/user.graphql

```diff
@@ -5,6 +5,7 @@ type User {
   firstName: String!
   middleName: String
   lastName: String!
+  status: String!
```

Comment:
Can we make use of [Graphql Enum](https://graphql.org/learn/schema/#enumeration-types) to ensure this is a well defined value and not some random string?